### DISCUSSION TITLE
is there a way i can run the site locally?

### DISCUSSION THREAD

**Discussion by cvssn** [monkeytypegame/monkeytype#3793] [Questions]
Is there any way I can download the source code and run the site locally? Would i have to install the dependencies? Is it possible to do this with just what is offered in the github repo?

**Comment by JG606** [comment]
I haven't done so yet for myself - but looks like there are some instructions here that would be a good starting point :)
https://github.com/monkeytypegame/monkeytype/blob/master/CONTRIBUTING_ADVANCED.md#prerequisites
